ui/raidboss/data/07-dt/raid/r5s.ts

```diff
@@ -134,6 +135,7 @@ const triggerSet: TriggerSet<Data> = {
         combo: {
           en: 'Start ${dir} (${num} hits) => ${mech}',
           cn: '${dir} 开始 (打 ${num} 次) => ${mech}',
+          ko: '${dir}으로 (${num}번 공격) => ${mech}',
```

Comment:
이 부분은 취향차이일수도 있지만,
```suggestion
          ko: '${dir} 시작 (${num}번 공격) => ${mech}',
```
처럼 원문 의미를 살려두는걸 선호합니다. `동쪽 시작 (3번 공격) => 힐러 그룹 쉐어`
아니다 싶으면 거절하셔도 됩니다...